_blog.yml

```
@@ -5813,4 +5813,17 @@
   date: April 5, 2025
   tags:
     - llama
-    - meta
\ No newline at end of file
+    - meta
+
+- local: leaderboard-3c3h-aragen-ifeval
+  title: "Arabic Leaderboards: Introducing Arabic Instruction Following, Updating AraGen, and More"
+  author: alielfilali01
+  guest: true
+  thumbnail: /blog/assets/leaderboards-on-the-hub/thumbnail_3c3h_aragen.png
+  date: April 6, 2025
```

```suggestion
  date: April 8, 2025
```

Let's release tomorrow maybe?